backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserAuthorizationVerifier.java

```diff
@@ -0,0 +1,33 @@
+package gov.cdc.usds.simplereport.config.authorization;
+
+import java.util.Optional;
+
+import gov.cdc.usds.simplereport.config.simplereport.AdminEmailList;
+import gov.cdc.usds.simplereport.service.OrganizationService;
+import gov.cdc.usds.simplereport.service.model.CurrentOrganizationRoles;
+import gov.cdc.usds.simplereport.service.model.IdentityAttributes;
+import gov.cdc.usds.simplereport.service.model.IdentitySupplier;
+
+public class UserAuthorizationVerifier {
+
+    private AdminEmailList _admins;
+    private IdentitySupplier _supplier;
+    private OrganizationService _orgService;
+
+    public UserAuthorizationVerifier(AdminEmailList admins, IdentitySupplier supplier, OrganizationService orgService) {
+        super();
+        this._admins = admins;
+        this._supplier = supplier;
+        this._orgService = orgService;
+    }
+
+    public boolean userHasSiteAdminRole() {
+        IdentityAttributes id = _supplier.get();
+        return id != null && _admins.contains(id.getUsername());
+    }
+
+    public boolean userHasOrgAdminRole() {
```

Comment:
is anything calling this? (even if no, no need to delete it, i don't think)

Reply:
Not yet, and probably not ever—we should replace this with a general permission-checker for org-level permissions, and call that from our SPeL annotations.